### ISSUE TITLE
storage should have byte_size() function

### ISSUE THREAD

**Issue by omry** [torch/torch7#176]
storage should have byte_size() function that will return how many bytes the storage is using.

**Comment by timharley** [comment]
Storages and Tensors now have `:elementSize()`, which combined with `:nElement()` for Tensors or `:size()` for Storages, should meet your needs?

**Comment by omry** [comment]
yes, but would be even better to add a convenience method to return byte size.
feel free to close if you disagree.

**Comment by soumith** [comment]
we used to have this as :footprint() in eblearn. If the name sounds okay, I can quickly implement it at the lua level.

**Comment by omry** [comment]
sizeBytes() is a more accurate name. 'footprint' is not very well defined.
